src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariantsIntegrationTest.java

```diff
@@ -0,0 +1,751 @@
+package org.broadinstitute.hellbender.tools.walkers.variantutils;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.IntegrationTestSpec;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+
+public class SelectVariantsIntegrationTest extends CommandLineProgramTest {
+
+    // For now we're retaining hardcoded paths to the test files on the broad server
+    // Any tests dependent on these files are in the testNG group "requiresRemoteAccess"
+    // To enable these tests (which require access to the  broad files), set remoteTestsEnabled=true
+    private static final boolean remoteTestsEnabled=false;
+
+    private static final String b36KGReference = "/humgen/1kg/reference/human_b36_both.fasta";
+    private static final String b37KGReference = "/humgen/1kg/reference/human_g1k_v37.fasta";
+    private static final String GATKDataLocation = "/humgen/gsa-hpprojects/GATK/data/";
```

Comment:
(or if there are only a few large vcfs, and they're not too huge, we could check them into `src/test/resources/large` using `git lfs`)